tools/instances2dict_with_polygons.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/python
```

Comment:
Can we add those two files in a subfolder inside `tools/`?

Reply:
updated